image/init.sh

```diff
@@ -20,6 +20,13 @@ fi
 
 trap stop_all HUP INT QUIT KILL TERM
 
+
+# initialize the db (pwd: admin) and create users
+sleep 5  # wait for PG to start...
```

Comment:
This is not the best solution, but the other solutions I found (e.g. http://stackoverflow.com/questions/35069027/docker-wait-for-postgresql-to-be-running ) seem a bit overkill.